### DISCUSSION TITLE
failed to open /dev/dri/renderD128: Permission denied

### DISCUSSION THREAD

**Discussion by superxii** [filecoin-project/lotus#9346] [Build and Sync] [answered]
When I started the lotus miner and worker, this error occur:
failed to open /dev/dri/renderD128: Permission denied

Looks like it is not related to lotus itself, but more related with permission stuff.
I am not sure will this affect the performance on lotus.

Can this be solved by simply changing the mode to 700?

**Comment by rjan90** [comment, marked as answer]
Hey @superxii!

The error itself won´t cause any issues when using Lotus, and will not effect performance any way. You can solve it by adding your user to the render usergroup 😄

**Reply by superxii** [reply]
Thanks!!!
